docs/input.md

```diff
@@ -151,13 +151,13 @@ add a label on top of the input (optional)
 
 |  Type  | Default |
 | :----: | :-----: |
-| string |  none   |
+| string **OR** React element or component |  none   |
 
 ---
 
 ### `labelStyle`
 
-styling for the label (optional)
+styling for the label (optional);  You only use this if `label` is a string
```

Comment:
```suggestion
styling for the label (optional);  You can only use this if `label` is a string
```

Reply:
Completely forgot about test! Thank you 🥇 and sorry for that